dm/master/server.go

```diff
@@ -149,16 +155,32 @@ func (s *Server) Start() error {
 			log.L().Error("fail to start gRPC server", zap.Error(err2))
 		}
 	}()
-	go InitStatus(httpL) // serve status
+
+	httpmux := http.NewServeMux()
+	HandleStatus(httpmux) // serve status
+
+	err = s.HandleHTTPApis(ctx, httpmux) // server http api
+	if err != nil {
+		return errors.Trace(err)
+	}
+
+	go func() {
```

Comment:
do we need to call `wg.Add(1)` for this goroutine (may also for the goroutine in L152)?

Reply:
https://github.com/pingcap/dm/pull/224/commits/8a46d0ab5b675ce8d59f671526cc4b68fd4b1803